webots_ros2_core/README.md

```diff
@@ -0,0 +1,127 @@
+# webots_ros2_core
```

Comment:
Note that some information are already available on the wiki for the released part, it would be nice to integrate them here too so that when we release the new version of the package we can simply copy paste and overwritten the current content of the wiki page: http://wiki.ros.org/webots_ros2/Tutorials/webots_ros2_core

Reply:
So, idea is to copy documentation from wiki.ros.org now, keep improving it here and periodically sync the changes with wiki.ros.org?

Having documentation on Github would be useful indeed since index.ros.org takes the docs from Git repos, e.g.:
https://index.ros.org/p/webots_ros2_epuck/github-cyberbotics-webots_ros2/#foxy

Reply:
Yes that sounds to be a good workflow especially for now that several packages are not released.